Gordon360/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+namespace Gordon360.Extensions.System;
+
+static public class DateTimeExtensions
+{
+    /// <summary>
+    /// Helper method that casts given date time to type UTC without changing the time
+    /// </summary>
+    /// <param name="dateTime"></param>
+    /// <returns>dateTime with type UTC</returns>
+    static public DateTime SpecifyUtc(this DateTime dateTime) => DateTime.SpecifyKind((DateTime)dateTime, DateTimeKind.Utc);
+    
+
+    /// <summary>
+    /// Function overload for nullable datetime
```

Comment:
Specify the given DateTime? as UTC, without changing the time value.

Reply:
I somehow thought that overload functions would inherit other comments. I'll add that in.